### ISSUE TITLE
AMP story Player: the player.add() function makes the player to load the 2nd story instead of the first

### ISSUE THREAD

**Issue by rnkimani** [ampproject/amphtml#30213]
## What's the issue?
- Instead of loading the first story, it loads the 2nd story and then while navigating back the first story is being loaded correctly. 

## How do we reproduce the issue?
- Add stories to player using player.add function

## What browsers are affected?
- All

## Which AMP version is affected?
- Latest
- Ref [#28416](https://github.com/ampproject/amphtml/issues/28416)

cc @Enriqe @newmuis

**Comment by Enriqe** [comment]
Hmm, I'm not able to repro. Can you provide a bit more context on your use case? are you using `add()` when there are no stories on the player, or are there some already?

**Comment by rnkimani** [comment]
When there are no stories.

**Comment by rnkimani** [comment]
Try these stories:
```
['https://stories.tappable.co/the-diversification-of-television?story', 
'https://stories.tappable.co/taps-kitchen?story',
 'https://stories.tappable.co/bbc-micro?story', 
'https://stories.tappable.co/our-world-from-space?story']
```

**Comment by rnkimani** [comment]
I have reproduced the error using svelte [here](https://svelte.dev/repl/22524365295644adbf4f154436af1724?version=3.25.0)

I might also be doing something wrong here, but you can see on the console the errors and the loaded story isn't the first one.

**Comment by Enriqe** [comment]
Thanks for the demo and the context. I found a nasty bug in our CSS that is surfaced specifically when creating an exit control. I will send a fix here #30236.
